lib/membrane_rtc_engine/endpoints/rtsp_endpoint.ex

```diff
@@ -38,19 +57,77 @@ if Enum.all?(
       state = %{
         rtc_engine: opts.rtc_engine,
         source_uri: opts.source_uri,
-        track: nil
+        stream_id: nil,
+        track_data: nil,
+        video: nil,
+        ssrc: nil
       }
 
-      structure = []
+      structure = [
+        child(:udp_source, %Membrane.UDP.Source{
+          local_port_no: @rtp_port,
+          recv_buffer_size: 500_000
```

Comment:
We can use default value, can't we? I wouldn't expect such big UDP datagrams even on localhost